### ISSUE TITLE
Add links to documentation for Mix, Compose and Fill

### ISSUE THREAD

**Issue by PoignardAzur** [linebender/peniko#22]
The concepts behind the `Mix`, `Compose` and `Fill` types are well-known in computer graphics, but they're not intuitive for a newcomer and hard to explain through text. (And even an experienced developer might not remember which is which.)

We should probably edit their respective doc comments to add links to pages illustrating the underlying concepts. ~~We can probably find good resources on MDN or Wikipedia.~~ (EDIT: So far I haven't found any.)

**Comment by nicoburns** [comment]
+1 for this. It might also be useful to list 2 or 3 "sensible default" values (like `Normal` and/or `Clip` and their use cases).

**Comment by PoignardAzur** [comment]
Is there a mirror to that spec somewhere? The images of that link won't load.

**Comment by PoignardAzur** [comment]
> Older spec seems to work better: https://www.w3.org/TR/compositing-1/

Yeah, I think we should steal basically everything from the document: text (we already do), formulas and visual aids.

If we do so we will have to add the W3 software license in the repo: https://www.w3.org/copyright/software-license-2023/
